src/main/java/view_comment/drivers/ViewCommentDatabaseAccess.java

```diff
@@ -0,0 +1,62 @@
+package view_comment.drivers;
+
+import com.opencsv.exceptions.CsvException;
+import view_comment.use_case.ViewCommentDsGateway;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+
+
+public class ViewCommentDatabaseAccess implements ViewCommentDsGateway {
```

Comment:
If Regan's branch with the Data Access gets merged before this one, let's put all of this in that collective DataAccess

Reply:
ok